EORTC/load_stage.sql

```diff
@@ -0,0 +1,1146 @@
+/**************************************************************************
+* Copyright 2016 Observational Health Data Sciences AND Informatics (OHDSI)
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* Authors: Maria Rogozhkina,
+* Aliaksei Katyshou, Vlad Korsik,
+* Alexander Davydov, Oleg Zhuk, EORTC QLQ Group
+* Date: 2024
+**************************************************************************/
+
+--Update latest_update field to new date
+DO
+$_$
+    BEGIN
+        PERFORM VOCABULARY_PACK.SetLatestUpdate(
+                pVocabularyName => 'EORTC QLQ',
+                pVocabularyDate => (SELECT TO_DATE(TO_CHAR(qs.updatedate, 'YYYY-MM-DD'), 'YYYY-MM-DD')
+                                    FROM sources.eortc_questionnaires qs
+                                    WHERE qs.name = 'Core'
+                                    ORDER BY TO_DATE(TO_CHAR(qs.updatedate, 'YYYY-MM-DD'), 'YYYY-MM-DD') DESC
+                                    LIMIT 1),
+                pVocabularyVersion => (SELECT 'EORTC QLQ defined by ' || qs.name || ' version ' || TO_CHAR(qs.updatedate, 'YYYY_MM')
+                                       FROM sources.eortc_questionnaires qs
+                                       WHERE qs.name = 'Core'
+                                       ORDER BY TO_DATE(TO_CHAR(qs.updatedate, 'YYYY-MM-DD'), 'YYYY-MM-DD') DESC
+                                       LIMIT 1),
+                pVocabularyDevSchema => 'DEV_EORTC'
+            );
+    END
+$_$;
+
+--Truncate all working tables
```

Comment:
Please add standard TRUNCATE to all tables, even if not a drug vocab is processed